### ISSUE TITLE
[BUG] Setting the background color to transparent results in black background

### ISSUE THREAD

**Issue by gerken-tss** [the-airbender/quill_html_editor#34]
**Describe the bug**
When I set the property `backgroundColor` property of `QuillHtmlEditor` to `Colors.transparent`, because I might want it to have the background color of the underlying Widget, it displays the editor with a black background color instead of a transparent one.

I was able to reproduce this on:
- iPhone with iOS 16.2
- iOS Simulator
- Android simulator

When I rendered the content of the `_getQuillPage()` function into a local HTML file without the Flutter variables and callbacks, it is displayed with a transparent background which leads me to the conclusion that it's either caused by [webviewx](https://pub.dev/packages/webviewx) or [webview_flutter](https://github.com/flutter/plugins/tree/master_archive/packages/webview_flutter).

**Screenshots**

<img width="255" alt="Screenshot 2023-03-09 at 14 15 29" src="https://user-images.githubusercontent.com/84321030/224034407-fe31cfdd-edb5-4825-b4e5-401258acad28.png">

**Flutter Doctor**
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.7.3, on macOS 13.1 22C65 darwin-arm64, locale
    en-DE)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 14.2)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.3)
[✓] IntelliJ IDEA Ultimate Edition (version 2022.2.3)
[✓] Connected device (5 available)
[✓] HTTP Host Availability

• No issues found!
```

**Additional context**
 In general, it might be a good idea to switch to [flutter_inappwebview](https://pub.dev/packages/flutter_inappwebview) as `flutterx` is not maintained anymore and we will not expect any fixes, I guess.

**Comment by the-airbender** [comment]
Hi @gerken-tss 

Turns out the issue is not with webview. The `toHex` method that converts color to hex code is converting the transparent color to black. I have resolved this issue. Will move the fix along with other bugs probably in the weekend. 

Also, `webviewx` package is a wrapper for `webview_flutter` which is one of flutter's packages.

You are correct the best possible alternative for webview_flutter is `flutter_inappwebview` which is an amazing package. 

Currently we are using the latest webview flutter packages and everything seems working well.
I appreciate your suggestion, will consider it in future if required.

Thanks :)

**Comment by the-airbender** [comment]
Hi @gerken-tss ,

This issue is resolved, please check the ver [2.1.6](https://pub.dev/packages/quill_html_editor/versions/2.1.6)

Thanks!

**Comment by gerken-tss** [comment]
@the-airbender 

Thank you very much for trying to fix it. 

However, I fear it's not working. Neither in an app in which I'm using it, nor in the example of your project.

If I wrap `QuillHtmlEditor` with a `Container` that has a `backgroundColor` and a `padding`, it doesn't show (in this case I tried green):

<img width="260" alt="Screenshot 2023-03-16 at 09 24 56" src="https://user-images.githubusercontent.com/84321030/225557740-7271b1aa-1291-4a5a-acf1-2586e2d1c896.png">

It seems like setting the `background-color` property of the elements inside the DOM doesn't have any effect. Even if I hardcode this:

```css
* {
   background-color: transparent !important;
}
```

in the `style` tag, nothing happens.

My guess is that it's because `webviewx` is very outdated. It uses `webview_flutter` [`2.0.13`](https://github.com/adrianflutur/webviewx/blob/main/pubspec.yaml) although the most recent version is [`4.0.6`](https://pub.dev/packages/webview_flutter/changelog).

Like I said above I think we're hitting boundaries with using `webviewx` here. What goes against just using `webview_flutter` directly or switching to `flutter_inappwebview`?
